src/components/CollectionShare.tsx

```diff
@@ -33,16 +37,82 @@ interface Props {
 interface formValues {
     email: string;
 }
+
+interface passFormValues {
+    linkPassword: string;
+}
 interface ShareeProps {
     sharee: User;
     collectionUnshare: (sharee: User) => void;
 }
 
+const style = {
+    ...SelectStyles,
+    dropdownIndicator: (style) => ({
+        ...style,
+        margin: '0px',
+    }),
+    singleValue: (style) => ({
+        ...style,
+        margin: '0px',
+        backgroundColor: '#282828',
+        color: '#d1d1d1',
+        display: 'block',
+        width: '120px',
+    }),
+    control: (style, { isFocused }) => ({
+        ...style,
+        backgroundColor: '#282828',
+        margin: '0px',
+        color: '#d1d1d1',
+        borderColor: isFocused ? '#51cd7c' : '#444',
+        boxShadow: 'none',
+        ':hover': {
+            borderColor: '#51cd7c',
+            cursor: 'text',
+            '&>.icon': { color: '#51cd7c' },
+        },
+    }),
+};
+
+const f = (i) => {
+    return { label: i.toString(), value: i };
+};
+
+const expiryOptions = [
+    { label: 'never', value: () => 0 },
+    {
+        label: 'after 1 hour',
+        value: () => DateTime.utc().plus({ hours: 1 }).toMillis() * 1000,
+    },
+    {
+        label: 'after 1 day',
+        value: () => DateTime.utc().plus({ days: 1 }).toMillis() * 1000,
+    },
+    {
+        label: 'after 1 week',
+        value: () => DateTime.utc().plus({ days: 7 }).toMillis() * 1000,
```

Comment:
this can be just do as `new Date(Date.now() + 7 * A_DAY_IN_MICROSEC)` na 

luxon is a very heavy library 😬 . we should avoid using it